src/tests/unit/tests/DetailsView/components/export-dialog.test.tsx

```diff
@@ -52,35 +49,30 @@ describe('ExportDialog', () => {
         const isOpenOptions = [true, false];
 
         it.each(isOpenOptions)('with open %p', isOpen => {
-            provideBlobMock
-                .setup(p => p(It.isAny(), It.isAnyString()))
-                .returns(() => blobStub)
-                .verifiable(Times.once());
-            windowUtilsMock
-                .setup(w => w.createObjectURL(blobStub))
+            props.isOpen = isOpen;
+            fileProviderMock
+                .setup(f => f.provideURL(It.isAny(), It.isAnyString()))
                 .returns(() => 'fake-url')
                 .verifiable(Times.once());
-            props.isOpen = isOpen;
             const wrapper = shallow(<ExportDialog {...props} />);
             expect(wrapper.getElement()).toMatchSnapshot();
+
+            fileProviderMock.verifyAll();
         });
     });
     describe('user interaction', () => {
         it('closes the dialog onDismiss', () => {
             onCloseMock.setup(oc => oc()).verifiable(Times.once());
-            provideBlobMock
-                .setup(p => p(It.isAny(), It.isAnyString()))
-                .returns(() => blobStub)
-                .verifiable(Times.once());
-            windowUtilsMock
-                .setup(w => w.createObjectURL(blobStub))
+            fileProviderMock
+                .setup(f => f.provideURL(It.isAny(), It.isAnyString()))
```

Comment:
`f` inside the `setup` call is a little short for a name, how does `provider` sounds?